test/readme.md

```diff
@@ -1,110 +1,268 @@
 
-```yaml
-SINGLE_MONGOS_LB_URI: "mongodb://127.0.0.1:8000/?loadBalanced=true"
-MULTI_MONGOS_LB_URI: "mongodb://127.0.0.1:8001/?loadBalanced=true"
-```
+1. In a terminal, navigate to your node driver directory:
 
-Since it's a flat yaml file, you can run the following to get a sourceable environment file:
+   `cd node-mongodb-native`
+1. Use the Evergreen `patch` command. `-y` skips the confirmation dialog. `-u` includes uncommitted changes. `-p [project name]` specifies the Evergreen project. --browse opens the patch URL in your browser.
 
-```sh
-cat lb-expansion.yml | sed 's/: /=/g' > lb.env
-```
+   `evergreen patch -y -u -p mongo-node-driver --browse`
+1. In your browser, select the build variants and tasks to run.
 
-You have to add an additional environment variable to the end of this `lb.env` file:
+## Using a Pre-Release Version of a Dependent Library
 
-```sh
-FAKE_MONGODB_SERVICE_ID="true"
-```
+You may want to test the driver with a pre-release version of a dependent library (e.g., [bson][js-bson]).
+Follow the steps below to do so.
+1. Open [package.json](../package.json)
+1. Identify the line that specifies the dependency
+1. Replace the version number with the commit hash of the dependent library. For example: `"bson": "e29156f7438fa77c1672fd70789d7ade9ca65061"`
+1. Run `npm install` to install the dependency
 
-This enables logic in the driver to stick in a fake service id on responses since that's what a real LB deployment is required to do.
-With those variables sourced, you can run the whole test suite as you normally would.
+Now you can run the automated tests, run manual tests, or kick off an Evergreen build from your local
+repository.
 
-> Please note, `FAKE_MONGODB_SERVICE_ID` will no longer be needed with the completion of [NODE-3431](https://jira.mongodb.org/browse/NODE-3431).
+## Manually Testing the Driver
 
-```sh
-npm run check:test
-```
+You may want to manually test changes you have made to the driver. The steps below will walk you through how to create a new Node project that uses your local copy of the Node driver.  You can
+modify the steps to work with existing Node projects.
 
-Take note of the `[ topology type: load-balanced ]` printout from mocha to make sure it picked up the environment as expected.
+1. Navigate to a new directory and create a new Node project by running `npm init` in a terminal and    working through the interactive prompts. A new file named `package.json` will be created for you.
+1. In `package.json`, create a new dependency for `mongodb` that points to your local copy of the driver. For example:
+   ```
+   "dependencies": {
+     "mongodb": "/path-to-your-copy-of-the-driver-repo/node-mongodb-native"
+   }
+   ```
+1. Run `npm install` to install the dependency.
+1. Create a new file that uses the driver to test your changes. See the [MongoDB Node.js Quick Start Repo][node-quick-start] for example scripts you can use.
```

Comment:
I'd add something like, "Note when making driver changes you will have to run `npm run build:ts` with each change to have it take effect".

Reply:
Great catch! Thank you!